internal/pkg/template/manifest/manifest.go

```diff
@@ -24,9 +30,25 @@ func New() *Manifest {
 	}
 }
 
-func Load(fs filesystem.Fs, jsonNetCtx *jsonnet.Context) (*Manifest, error) {
-	// Load file content
-	content, err := loadFile(fs, jsonNetCtx)
+// Load manifest File.
+func Load(fs filesystem.Fs) (*File, error) {
+	path := Path()
+	if !fs.IsFile(path) {
+		return nil, fmt.Errorf("manifest \"%s\" not found", path)
+	}
+
+	f, err := fs.ReadFile(filesystem.NewFileDef(path).SetDescription("manifest"))
+	if err != nil {
+		return nil, err
+	}
+
+	return &File{file: f}, nil
+}
+
+// Evaluate Jsonnet content.
+func (f *File) Evaluate(jsonNetCtx *jsonnet.Context) (*Manifest, error) {
```

Comment:
Zvolil som nazov `Evaluate` ako je v `jsonnet` kniznici: https://github.com/google/go-jsonnet/blob/f699b0ea4251b6c63b7b9860b5a6e9c68a4f4aef/vm.go#L179

Minule sme risili ci `parse/execute/evaluate`, dava mi to asi najvacsi zmysel:
https://stackoverflow.com/a/38636647